src/lib/monitor.ts

```
@@ -90,16 +92,25 @@ export async function monitor(
     targetFile?: string,
     ): Promise<MonitorResult> {
   apiTokenExists();
+
+  const packageManager = meta.packageManager;
+  analytics.add('packageManager', packageManager);
+  analytics.add('isDocker', !!meta.isDocker);
+
+  if (meta['experimental-dep-graph'] && packageManager === 'npm') {
```

Shouldn't we somehow verified FF via `${registry API}/cli-config/feature-flags/` call?

---

It's verified at this point already

https://github.com/snyk/snyk/blob/master/src/cli/commands/monitor.ts#L87

---

Cool! Fail fast, I love it!